index.bs

```diff
@@ -4528,7 +4528,9 @@ strong [=/map=] from handle ids to their corresponding objects.
 
 <pre class="cddl remote-cddl local-cddl">
 script.LocalValue = (
```

Comment:
It makes me wonder whether `LocalValue` should be renamed to `ArgumentValue` now (as it is effectively used as an argument).

Reply:
LocalValue derives its name from the spec instructions.